windows-container-samples/ASP-NET-Blog-Application/README

```diff
@@ -0,0 +1,65 @@
+
+# Run blog application with Docker Compose:
+# ----------------------------------------------------
+
+# Obtain prerequisite container images:
+microsoft/windowsservercore
+microsoft/mssql-server-2014-express-windows
+
+# Build blog database image; run the following from .\db
+docker build -t blogdb .
```

Comment:
You can use `docker-compose build` to do this. Your compose file specifies `image` instead of using `build`, which I think would be simpler